solidity/contracts/deposit/DepositUtils.sol

```diff
@@ -66,6 +66,9 @@ library DepositUtils {
         uint256 fundedAt; // timestamp when funding proof was received
         bytes utxoOutpoint;  // the 36-byte outpoint of the custodied UTXO
 
+        /// @notice Map of ETH balances an address can withdraw after contract reaches ends-state.
```

Comment:
Why a `@notice` here when we don't do it anywhere else?

Reply:
we do it in `approvedDigests` mapping right below. We can in-line both instead

Reply:
I'm down either way---if this results in proper API doc JSON that's fine, though we should use `@dev` rather than `@notice` given I don't think we expect users to directly interact with the mapping.